### DISCUSSION TITLE
[SpeechlyClient] Websocket closed due to error

### DISCUSSION THREAD

**Discussion by rcolepeterson** [speechly/speechly#314] [General]
Has anyone seen this? All my apps that use speechly are firing this. The code for these apps have not changed in over 8 months.

code: 1011
reason: "the server has encountered an internal error"
wasClean: true

I feel like something was changed on the speechly side.

**Comment by tai-michael** [comment]
Getting the same error.

**Comment by jermlab** [comment]
Same.

**Comment by jannepy** [comment]
Sorry for the inconveniences! We have deprecated some older ASR models, if your app has not been deployed recently it may be referring to a model that is no longer available. Please redeploy your app and try again. In the dashboard, you may need to insert a dummy change into the configuration, e.g. a newline, to activate the deploy-button.

**Reply by jermlab** [reply]
Thank you for the prompt reply.
I redeployed and the app is now working again, but the lag is considerably longer now. Using the preview feature in the dashboard has about 2.2 - 2.4 seconds between the end of speech and the first response. Before it was less than 1 second. 

The accuracy seems much worse than before as well. Saying "cat" gave the responses [yes, yeah, okay, kat, yes, okay, kit, kit, cat, katie, kit]

My app can't be used with this delay and accuracy. Do you have any suggestions?

**Reply by jannepy** [reply]
The default model in the dashboard is dictation-oriented model which indeed has a higher latency. For low-latency use cases, please select "large-lowlatency" model in the drop-down menu.

**Reply by jermlab** [reply]
Thank you! I should have checked more carefully.

**Reply by rcolepeterson** [reply]
THANK YOU!

**Reply by amabes** [reply]
How do you plan to notify customers when you need to make changes like this in the future?
What steps have you taken to prevent customers from experiencing outages like this in the future?
